### PR TITLE
Update report subscribers

### PR DESCRIPTION
**Pull request by Fosol**
Replacing FlexTable component with new light weight Grid component.

## Editor Report Subscriber Admin

image

### CODE REVIEW

File: app/editor/src/components/grid/styled/Grid.tsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+
+interface IGridProps {
+  columns: (IGridHeaderColumnProps | undefined)[];
+}
+
+export const Grid = styled.div<IGridProps>`
   .grid-table {
     display: grid;
-    grid-template-columns: 30px 1fr 1fr 1fr 1fr;
+    grid-template-columns: ${(props) => props.columns.map((col) => col?.size ?? '1fr').join(' ')};
```
Comment (Fosol) [inline app/editor/src/components/grid/styled/Grid.tsx:12]: We can now support dynamic column sizes.

File: app/editor/src/features/admin/notifications/NotificationSubscribersForm.tsx [hunk 2]
```diff
@@ -3,21 +3,30 @@ import { SortDirection } from 'components/grid/SortAction';
...
-import { CellEllipsis, Checkbox, INotificationModel } from 'tno-core';
+import { CellEllipsis, Checkbox, INotificationModel, IUserFilter } from 'tno-core';
 
 import { NotificationFilter } from './NotificationFilter';
 
 export const NotificationSubscribersForm = () => {
   const { values, setFieldValue } = useFormikContext<INotificationModel>();
   const [{ users }, { findUsers }] = useUsers();
 
+  const [filter, setFilter] = React.useState<IUserFilter>({ page: 1, quantity: 100, sort: [] });
```
Comment (Fosol) [inline app/editor/src/features/admin/notifications/NotificationSubscribersForm.tsx:14]: Fixed filtering issues

File: app/editor/src/features/admin/reports/ReportFormSubscribers.tsx [hunk 3]
```diff
@@ -48,28 +64,83 @@ export const ReportFormSubscribers: React.FC = () => {
           await findUsers({ page: 1, quantity: users.quantity, keyword: value });
         }}
       />
-      <FlexboxTable
```
Comment (Fosol) [inline app/editor/src/features/admin/reports/ReportFormSubscribers.tsx:51]: Replaced with Grid

File: app/editor/src/components/grid/styled/Grid.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/editor/src/components/grid/styled/Grid.tsx:12]: Nice

**Review by jdtoombs — APPROVED** [review]

**Pull request merged**